pkg/volume/cephfs/cephfs.go

```diff
@@ -335,8 +339,8 @@ func (cephfsMounter *cephfsMounter) checkFuseMount() bool {
 	execute := cephfsMounter.plugin.host.GetExec(cephfsMounter.plugin.GetPluginName())
 	switch runtime.GOOS {
 	case "linux":
-		if _, err := execute.Run("/usr/bin/test", "-x", "/sbin/mount.fuse.ceph"); err == nil {
-			glog.V(4).Info("/sbin/mount.fuse.ceph exists, it should be fuse mount.")
+		if _, err := execute.Run("/usr/bin/ceph-fuse", "--version"); err == nil {
```

Comment:
@zhangxiaoyu-zidif i think this is what in the first ceph-fuse, what happened since then?

Reply:
In some distro, the path is /usr/sbin/mount.fuse.ceph. I think if we use ceph-fuse, we should just check it.

Reply:
Did some searching, found that the first version uses `/sbin/mount.fuse.ceph`  too, see https://github.com/kubernetes/kubernetes/pull/55866/files#diff-021042d70c9aac8994638a8d0827b1e0R330.

It's better to check `/usr/bin/ceph-fuse` because we execute `/usr/bin/ceph-fuse` command instead of `/sbin/mount.fuse.ceph` in `execFuseMount` function. It's more consistent.

If we require `ceph-fuse` must be at `/usr/bin/ceph-fuse`, I suggest to use absolute path `/usr/bin/ceph-fuse` instead of `ceph-fuse` in [`execFuseMount`](https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/cephfs/cephfs.go#L412).

Reply:
Or just use `ceph-fuse` directly here? for the case `ceph-fuse` is not installed standardly.

Reply:
I'm fine with either way. cc @rootfs @zhangxiaoyu-zidif